src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -81,22 +108,7 @@ export default class FieldBlocks extends React.Component<FieldTypeProps<Array<Bl
 
     // eslint-disable-next-line no-unused-vars
     renderCollapsedBlockContent = (value: Object, type: string, index: number) => {
-        if (!type) {
-            throw new Error(
-                'It is impossible that a collapsed block has no type. This should not happen and is likely a bug.'
-            );
-        }
-
-        const {formInspector, schemaPath} = this.props;
-        const blockSchemaTypes = formInspector.getSchemaEntryByPath(schemaPath).types;
```

Comment:
@danrot do you know why here was used `formInspector.getSchemaEntryByPath(schemaPath).types` instead of `this.props.types`?